### ISSUE TITLE
Y2 Unsecured

### ISSUE THREAD

**Issue by gbrown128** [marqs85/ossc_pcb#8]
Hi Marqs,
On my unit Y2 (audio ADC crystal) was installed with quite a lot of lead length between it and the board. It had flopped over and was worryingly close to touching the back of the SCART connector pins.

Depends on your manufacturing process, but I'd suggest swapping to a horizontal footprint with a solder pad to tie the crystal can to the PCB, or adding the wire loop to brace the crystal in place, or just some super glue, as I've used to fix mine!

 Otherwise very pleased with the design - thank you very much!

**Comment by marqs85** [comment]
Where did you buy your board from? The crystal should lie horizontally without too much extra lead length. A different footprint could indeed be a better solution even though so far I've not heard anyone else having such issues. SMD footprint would most compact and robust, but it's not ideal when DIY aspect is considered.

**Comment by gbrown128** [comment]
Board came from videogameperfection.com  Unfortunately I neglected to grab a picture before gluing it down, but that's the amount of lead length present on my unit.

It's possible my unit just had a rather rough ride in shipping, but the crystal was bent over about 45 degrees in all three axes. To be fair it is quite possible it would've never caused an issue. I just happened to take the lid off my unit out of curiosity and the crystal caught my eye! Given this unit will travel a lot I thought it best to glue it down!

![photo_2019-10-28_15-17-30](https://user-images.githubusercontent.com/4569238/67691306-3ddfd280-f996-11e9-9187-3d6f0c8481a1.jpg)
